### PR TITLE
Introduce icon-v2 option type

### PR DESCRIPTION
**Pull request by andreiglingeanu**
Hello. This pull request will introduce a great new option type: `icon-v2`.

First, [a](https://i.imgur.com/TbcqVbW.png) [couple](https://i.imgur.com/bkfARU1.png) [of](https://i.imgur.com/2dyGlGA.png) [screenshots](https://i.imgur.com/zAjqE4X.png).

First, the usage, which has a couple of [defaults](https://github.com/ThemeFuse/Unyson/blob/1764247041e81e32f7c099a742ec3c1dc3c3187e/framework/includes/option-types/icon-v2/class-fw-option-type-icon-v2.php#L147): 
```php
'icon-1'    => array(
    'label' => __( 'Logo', 'unyson' ),
    'desc'  => __( 'Desc', 'unyson' ),
    'type'  => 'icon-v2'
)
```

The other thing that may be useful to the theme developers may be the [filters](https://github.com/ThemeFuse/Unyson/blob/1764247041e81e32f7c099a742ec3c1dc3c3187e/framework/includes/option-types/icon-v2/includes/class-fw-icon-v2-packs-loader.php#L88) it [listens](https://github.com/ThemeFuse/Unyson/blob/1764247041e81e32f7c099a742ec3c1dc3c3187e/framework/includes/option-types/icon-v2/includes/class-fw-icon-v2-packs-loader.php#L255) to. More details about them right in the [source](https://github.com/ThemeFuse/Unyson/blob/1764247041e81e32f7c099a742ec3c1dc3c3187e/framework/includes/option-types/icon-v2/includes/class-fw-icon-v2-packs-loader.php#L25).

Any comments and suggestions are welcome.

### CODE REVIEW

**Comment by moldcraft** [comment]
`entypo`, `linecons`, `typcn` (all fonts) should be [here](https://github.com/ThemeFuse/Unyson/tree/master/framework/static/libs)

File: framework/includes/option-types/icon-v2/class-fw-option-type-icon-v2.php [hunk 1]
```diff
@@ -0,0 +1,174 @@
+<?php
+
+if (! defined('FW')) { die('Forbidden'); }
+
+require_once __DIR__ . '/includes/class-fw-icon-v2-packs-loader.php';
+require_once __DIR__ . '/includes/class-fw-icon-v2-favorites.php';
```
Comment (moldcraft) [inline framework/includes/option-types/icon-v2/class-fw-option-type-icon-v2.php]: PHP 5.2 compatibility, replace `__DIR__` with `dirname(__DIR__)`

https://travis-ci.org/ThemeFuse/Unyson/jobs/137823262#L205

File: bin/load-latest-fonts.php [hunk 2]
```diff
@@ -6,22 +6,11 @@
...
-			'fonts/FontAwesome.otf',
-			'fonts/fontawesome-webfont.eot',
-			'fonts/fontawesome-webfont.svg',
-			'fonts/fontawesome-webfont.ttf',
-			'fonts/fontawesome-webfont.woff',
-			'fonts/fontawesome-webfont.woff2'
-		)
-	),
+function get_libs_dir() {
+	return __DIR__ . '/../framework/static/libs/';
```
Comment (moldcraft) [inline bin/load-latest-fonts.php]: Again `__DIR__` -> `dirname(__FILE__)`

Comment (andreiglingeanu) [inline bin/load-latest-fonts.php]: This script should be run by ourselves from time to time, in order to keep packs versions up to date. It is not included in the runtime.

**Comment by moldcraft** [comment]
Thank you for sharing your work!

I hope it fixes all complains that users have on v1 and everyone will be happy.

**Comment by andreiglingeanu** [comment]
@moldcraft You're welcome. Yeah, also hope so. Anyway, we'll have already have a solution that's a bit more flexible than the previous one.

By the way, will you handle documentation by yourself?

**Comment by moldcraft** [comment]
Yes

**Pull request merged**